src/test/java/htsjdk/tribble/readers/TabixReaderTest.java

```diff
@@ -95,7 +98,15 @@ public void testIterators() throws IOException {
         iter=tabixReader.query("1",Integer.MAX_VALUE-1,Integer.MAX_VALUE);
         Assert.assertNotNull(iter);
         Assert.assertNull(iter.next());
-        
+
+        iter=tabixReader.query("1",-1,Integer.MAX_VALUE);
+        Assert.assertNotNull(iter);
+        Assert.assertNull(iter.next());
+
+        iter=tabixReader.query("1",Integer.MAX_VALUE,-1);
+        Assert.assertNotNull(iter);
+        Assert.assertNull(iter.next());
+
```

Comment:
Can we add a test case in which end == 0?

Reply:
Also add test cases for the other overloads of `query()`, since they have difference interval conventions (the String version is 1-based closed, the 3-arg version is 0-based closed-open!)